feature-libs/asm/core/config/asm-config.ts

```diff
@@ -11,6 +11,9 @@ import { Config, OccConfig } from '@spartacus/core';
   providedIn: 'root',
   useExisting: Config,
 })
+/**
+ * @deprecated. Please use feature-libs/asm/root/asm-config.ts
```

Comment:
The deprecated message should use the import path instead of the file path.  The import path is what the customers need to use the newer/non-deprecated class.